src/pudl/output/pudltabl.py

```diff
@@ -241,12 +236,15 @@ def _agg_table_name(self, table_name: str) -> str:
                 table_name = table_name.replace("_AGG", "")
         return table_name
 
-    def _get_table_from_db(self, table_name: str, resource: Resource) -> pd.DataFrame:
+    def _get_table_from_db(
+        self, table_name: str, resource: Resource, update: bool = False
```

Comment:
Is it worth logging a warning here?

Reply:
A warning about what?

Reply:
Ah sorry that was vague. I think I meant adding a warning about `update` being ignored but retained for backwards compatibility.

Reply:
Added logging warning.